### ISSUE TITLE
Add missing methods from pastas.Project

### ISSUE THREAD

**Issue by dbrakenhoff** [pastas/pastastore#3]
While looking through the pastas.Project code, I saw some methods that were not yet implemented in pastastore:
- get_parameters
- get_statistics
- ...

This issue is a reminder to look through pastas.Project and check if any of those methods would be nice to include in pastastore.

**Comment by rubencalje** [comment]
I would like to be able to send someone my database, so I would like some kind of method to save a database to a file (like to_file in Project). This is probably difficult though.

**Comment by OnnoEbbens** [comment]
I think if you use the `pystore` connector, you can zip the directory that contains your pystore and send it to someone. I don't think you have to call a `to_file` method because any change to the pastastore is automatically saved in the pystore. Altough I am not sure about this.

**Comment by dbrakenhoff** [comment]
If you're using Pystore you should be able to just send over that directory. Creating a new PystoreConnector and pointing to that directory should allow the other person to continue with your data.

For MongoDB there is also the `mongodump` command that allows the user to export a collection to a gzip file. But this is a bit complicated (to me anyway) since the database is running inside a docker container. I'm sure it's possible to figure that out somehow.

But I agree it would be nice to have some generic way of sharing a database. But I'm not quite sure what the best method would be. Perhaps we should define some export method in the ConnectorUtil class that creates 3 directories:
- oseries:  store DataFrames as csv/json?
- stresses:  store DataFrames as csv/json?
- models: store as .pas
With the option of making it into a zipfile. Then we'd also need an import_zip() in the ConnectorUtil class. JSON might be the way to go, since the .pas files are JSON as well. I think with csv/json files the risk of having version issues between packages is low?

If anyone has any suggestions/ideas feel free to post them here.

**Comment by dbrakenhoff** [comment]
> I don't think you have to call a to_file method because any change to the pastastore is automatically saved in the pystore. Altough I am not sure about this.

Yes, this should be the case!

**Comment by dbrakenhoff** [comment]
Writing and reading zip archives added by 07803cdfab94909c3e73782da7371866993bae84.

Still somewhat experimental, but writing and reading the archive is working.

If you have a PastaStore object `pstore`:
```python
pstore.to_zip("test.zip")

new_conn = pastastore.DictConnector("new_connector")
new_pstore = pastastore.PastaStore.from_zip("test.zip", new_conn)
```

Still to be added:
- get_parameters
- get_statistics

**Comment by dbrakenhoff** [comment]
Closed by #12
